inference-engine/src/low_precision_transformations/src/network_helper.cpp

```diff
@@ -463,7 +463,17 @@ std::shared_ptr<ngraph::opset1::Multiply> NetworkHelper::optimizeMultipliesAfter
             }
 
             auto newInput = multiply->input_value(1 - constant1->output(0).get_target_inputs().begin()->get_index());
-            auto newConst = fold<opset1::Multiply>(constant1, constant2);
+            auto multiplyResult = fold<opset1::Multiply>(constant1, constant2);
+            {
+                // optimize constant shape: used in rfcn-resnet101-coco
+                const auto multiplyResultConstant = as_type_ptr<opset1::Constant>(multiplyResult);
+                if ((multiplyResultConstant != nullptr) && NetworkHelper::isScalarLike(multiplyResultConstant)) {
+                    multiplyResult = std::make_shared<opset1::Constant>(
+                            multiplyResultConstant->get_element_type(),
+                            Shape{},
+                            multiplyResultConstant->cast_vector<float>()[0]);
```

Comment:
To my mind it is better to use `multyplyResult = NetworkHelper::toScalar(multiplyResultConstant)`